.github/workflows/release.yml

```diff
@@ -23,7 +27,7 @@ jobs:
           node-version: 20
 
       - name: Update npm
-        run: npm i -g npm@10
+        run: npm i -g npm@latest
```

Comment:
do we still need this? might also want to bump `actions/setup-node` and `actions/checkout` to v6

Reply:
we need to have npm@11.5.0 or greater [per the docs](https://docs.npmjs.com/trusted-publishers#github-actions-configuration), doesn't actually _have_ to be `@latest`, but I just copied out of the docs